src/modules/Blend/Module.js

```diff
@@ -43,7 +41,8 @@ module.exports = function Dynamic(options, UI, util) {
           p.get(x, y, 0),
           p.get(x, y, 1),
           p.get(x, y, 2),
-          p.get(x, y, 3)
+          p.get(x, y, 3),
+          x, y
```

Comment:
If all the other parameters are on newlines, let's keep the last two on newlines too. Let's make the code more systematic and easy to read.